dynamic-layers/toradex-bsp-common/recipes-kernel/linux/device-tree-overlays-mainline_%.bbappend

```diff
@@ -0,0 +1 @@
+require ${@ 'recipes-kernel/linux/add-secboot-kargs-overlay.inc' if 'tdx-signed' in d.getVar('OVERRIDES').split(':') else ''}
```

Comment:
@drewmoseley Why changing from `add-secboot-kargs-overlay.inc` to `recipes-kernel/linux/add-secboot-kargs-overlay.inc`? Usually `include` and `require` directives are able to find files if the included files are in the same directory as the file that it is including it.

Also, because you renamed the file and changed it in the same commit, Git thinks you are removing the old file and adding the new one. With this, we lose the history of the file. If this change is really required, would you please separate it in two commits, one to move the file and another to change it. This way, we can recover the history of the file with `git log --follow`.

Thanks!

Reply:
@sergioprado I had to change the include path because the files are not in the same directory as the bbappend.  In this case the include file stays in the main `recipes-kernel/linux/` directory and is included with a full path.  Since the include file is used by both device-tree-overlays and device-tree-overlays-mainline I think this makes sense.

I opted to both move the file and edit it in one step to avoid potential breakage of `git bisect` but if you are ok with that I can redo it as 2 separate commits.

Reply:
> @sergioprado I had to change the include path because the files are not in the same directory as the bbappend. In this case the include file stays in the main recipes-kernel/linux/ directory and is included with a full path. Since the include file is used by both device-tree-overlays and device-tree-overlays-mainline I think this makes sense.

Hmm, that is right. You moved only the append file. Any reason for not moving the `add-secboot-kargs-overlay.inc` as well, so this change would not be required?

> I opted to both move the file and edit it in one step to avoid potential breakage of git bisect but if you are ok with that I can redo it as 2 separate commits.

Yeah, I understand that there is a trade-off. @rborn-tx do you have any thoughts on this?

Reply:
The include file is used by both `dynamic-layers/toradex-bsp-common/recipes-kernel/linux/device-tree-overlays-mainline_%.bbappend` and `dynamic-layers/toradex-nxp/recipes-kernel/linux/device-tree-overlays_%.bbappend` so I left it in the main `recipes-kernel/linux` folder so both could use it.

For this particular change, I can always do two commits; first with the change to the full path and second of the move.  That should preserve the "follow" history as well as the "git bisect"ability.  Let me know if you want me to modify it as such.

Reply:
Just to add my 2 cents. If possible I prefer to keep the follow history of the files. Though this isn't a hard requirement from me or anything, just a preference.

Reply:
Yeah, I also think splitting the PR in two commits to keep the history provides more benefits than costs.

@drewmoseley would you please split this PR in two commits and submit an updated version?

Reply:
@sergioprado,  this is done.